test/unit/mdc-toolbar/foundation.test.js

```diff
@@ -0,0 +1,676 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {assert} from 'chai';
+import td from 'testdouble';
+
+import {createMockRaf} from '../helpers/raf';
+import {verifyDefaultAdapter} from '../helpers/foundation';
+import {setupFoundationTest} from '../helpers/setup';
+import MDCToolbarFoundation from '../../../packages/mdc-toolbar/foundation';
+
+const {cssClasses, numbers} = MDCToolbarFoundation;
+
+suite('MDCToolbarFoundation');
+
+test('exports strings', () => {
+  assert.isOk('strings' in MDCToolbarFoundation);
+});
+
+test('exports cssClasses', () => {
+  assert.isOk('cssClasses' in MDCToolbarFoundation);
+});
+
+test('exports numbers', () => {
+  assert.isOk('numbers' in MDCToolbarFoundation);
+});
+
+test('defaultAdapter returns a complete adapter implementation', () => {
+  verifyDefaultAdapter(MDCToolbarFoundation, [
+    'hasClass', 'addClass', 'removeClass', 'registerScrollHandler',
+    'deregisterScrollHandler', 'registerResizeHandler', 'deregisterResizeHandler',
+    'getViewportWidth', 'getViewportScrollY', 'getOffsetHeight',
+    'getFlexibleRowElementOffsetHeight', 'notifyChange', 'setStyle',
+    'setStyleForTitleElement', 'setStyleForFlexibleRowElement',
+    'setStyleForFixedAdjustElement',
+  ]);
+});
+
+const setupTest = () => setupFoundationTest(MDCToolbarFoundation);
+const approximate = (value, expected, delta) => {
+  return Math.abs(value - expected) < delta;
+};
+
+const mockHandler = (foundation, mockAdapter, mockRaf) => {
```

Comment:
nit: maybe `createMockHandlers` is a better name here since a verb denotes a method where as a noun ("mockHandler") might be confused with an instance? Also handler**s** since we're creating more than one.